src/test/java/test/ContextPathTest.java

```diff
@@ -36,8 +36,14 @@ void test_context_path(Product product) throws Exception {
                 .assertHasValue("ATL_TOMCAT_CONTEXTPATH", "/" + product.name());
 
 
-        assertEquals(resources.getStatefulSet(
-                product.getHelmReleaseName()).getContainer().get("readinessProbe").get("httpGet").get("path").asText(),
-                "/" + product.name() + "/status");
+        if(product.name().equals("bamboo")) {
```

Comment:
I would extract it to a different test. Having a conditional based on the input is making the test more opaque.